AdyenTwint/TwintComponent.swift

```diff
@@ -0,0 +1,18 @@
+//
+// Copyright (c) 2023 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+@_spi(AdyenInternal) import Adyen
+import TwintSDK
+
+public final class TwintComponent {
```

Comment:
Is this missing the actual implementation?

Reply:
yes, this will contain the code. This PR is only for integration. There is no flow implementation in this.